.circleci/config.yml

```diff
@@ -36,6 +35,8 @@ jobs:
           name: Run tests
           command: |
             . venv/bin/activate
+            pip uninstall -y pytest
+            pip install 'pytest<8.1.0'
```

Comment:
Why is there an `uninstall` step here? Also, why pin at <8.1.0?

Reply:
Pytest has an issue with something and was failing. Reverting back to a lower version solved the issue.